OpenNefia.Content/UI/Element/List/UiPagedList.cs

```diff
@@ -20,6 +21,8 @@ public class UiPagedList<T> : UiList<T>, IUiPaged
         public int CurrentPage => _pageModel.CurrentPage;
         public int PageCount => _pageModel.PageCount;
 
+        public bool Muted;
```

Comment:
As above, make this into an argument to `SetPage()` (but not in `IPaged`)

Reply:
changed in b1407d8